### PR TITLE
Cp edit, datacenter, and cloud server stragglers

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-c37577a950/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6b77afc3ad/how-to/).

File: content/cloud-servers/create-an-image-of-a-general-purpose-v1-cloud-server.md [hunk 1]
```diff
@@ -1,58 +1,59 @@
...
-1.  Log into the Cloud Control Panel.
+1.  Log in to the [Cloud Control Panel](https://login.rackspace.com).
 
-2.  Locate the server you want to create an image of.
+2.  In the top navigation bar, click **Select a Product > Rackspace Cloud**.
 
-3.  Click the **Actions** gear icon to the left of the server name and
+3.  Select **Servers > Cloud Servers**.
+
+3.  Locate the server you want to create an image of.
```
Comment (stephamon) [inline content/cloud-servers/create-an-image-of-a-general-purpose-v1-cloud-server.md]: Locate the server for which you want to create an image.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-ffd811a4b9/how-to/). :bow:

**Pull request merged**